app/domain/importers/content_details.rb

```diff
@@ -13,7 +13,8 @@ def initialize(content_id, base_path)
 
   def run
     response = items_service.fetch_raw_json(base_path)
+    number_of_pdfs = Performance::Metrics::NumberOfPdfs.run_for_api_response(response)
```

Comment:
I'd rather not to have a method specific for a response type.  What about passing only the details?